tests/contest/contest/src/main.rs

```diff
@@ -123,6 +125,7 @@ fn main() -> Result<()> {
     tm.add_test_group(Box::new(mounts_recursive));
     tm.add_test_group(Box::new(intel_rdt));
     tm.add_test_group(Box::new(sysctl));
+    tm.add_test_group(Box::new(scheduler));
```

Comment:
Just comment out this line (don't remove) , and if the linter complains, add the ignore lint annotation to the place where it is complaining.